### PR TITLE
14.0 mig base geoengine

### PR DESCRIPTION
**Pull request by shurshilov**

### CODE REVIEW

**Comment by Freni-OSI** [comment]
Hi @shurshilov , can you please make this PR ready for review by passing all checks? Thanks

File: base_geoengine/static/description/index.html [hunk 1]
```diff
@@ -450,6 +450,7 @@ <h2><a class="toc-backref" href="#id7">Authors</a></h2>
 <div class="section" id="contributors">
 <h2><a class="toc-backref" href="#id8">Contributors</a></h2>
 <ul class="simple">
+  <li>Shurshilov Artem &lt;<a class="reference external" href="mailto:shurshilov.a@yandex.ru&#64;camptocamp.com">Shurshilov Artem&#64;eurodoo.com</a>&gt;</li>
```
Comment (yvaucher) [inline base_geoengine/static/description/index.html:453]: Please don't edit this file manually.

File: base_geoengine/static/src/js/views/geoengine/geoengine_controller.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /* ---------------------------------------------------------
  * Odoo base_geoengine
  * Contributor Yannick Vaucher 2018 Camptocamp SA
+ * Copyright 2021 Shurshilov Artem
```
Comment (yvaucher) [inline base_geoengine/static/src/js/views/geoengine/geoengine_controller.js:4]: A copyright to comment a line ?

File: base_geoengine/static/src/js/widgets/geoengine_widgets.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ odoo.define('base_geoengine.geoengine_widgets', function (require) {
     var AbstractField = require('web.AbstractField');
     var geoengine_common = require('base_geoengine.geoengine_common');
     var BackgroundLayers = require('base_geoengine.BackgroundLayers');
-    var registry = require('web.field_registry');
+    var field_registry = require('web.field_registry');
```
Comment (yvaucher) [inline base_geoengine/static/src/js/widgets/geoengine_widgets.js:17]: this change is unecessary

File: base_geoengine/static/src/js/views/form_renderer.js [hunk 4]
```diff
@@ -8,6 +8,7 @@ odoo.define('base_geoengine.form_renderer', function (require) {
         // Add the TabListener to geo_type widget so that _renderMap is called
         // when clicking on the new tab
         _updateView: function ($newContent) {
+            var self=this;
```
Comment (yvaucher) [inline base_geoengine/static/src/js/views/form_renderer.js:11]: self is not used in this method

**Comment by max3903** [comment]
@Urvisha-OSI Please rescue the branch and fix the issue. Please add the reference to the new PR here.

**Comment by Urvisha-OSI** [comment]
> rescue

@max3903 i don't have access of this branch

**Comment by max3903** [comment]
@Urvisha-OSI Please create a new one and migrate the module following the steps of:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

**Comment by Urvisha-OSI** [comment]
> @Urvisha-OSI Please create a new one and migrate the module following the steps of: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

@max3903 please check new PR https://github.com/OCA/geospatial/pull/316 of this PR

**Pull request closed without merging**